src/pages/scientificServices/pipelines/components/inputs/PipelineFloatInput.test.tsx

```diff
@@ -3,23 +3,15 @@ import userEvent from '@testing-library/user-event';
 import React from 'react';
 import { PipelineInput } from 'src/libs/ajax/teaspoons/teaspoons-models';
 
-import { PipelineFloatInput } from './PipelineFloatInput';
-
-jest.mock('src/pages/scientificServices/pipelines/utils/pipeline-input-utils', () => ({
-  INPUT_DESCRIPTIONS: {
-    floatInput: {
-      label: 'Enter a float value',
-      placeholder: 'Enter a number',
-      helpText: 'Must be a valid floating-point number.',
-      validationRegex: String.raw`^(0(\.\d*)?|1(\.0*)?|\.\d+)$`,
-    },
-  },
-}));
+import { PipelineFloatInput, validatePipelineFloatInput } from './PipelineFloatInput';
 
 const mockInput: PipelineInput = {
   name: 'floatInput',
   type: 'FLOAT',
   isRequired: true,
+  description: 'Must be a valid floating-point number.',
```

Comment:
are you deliberately testing the case where no displayName is provided (in which we fall back on the name) ?